src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationExecutor;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.impl.OperationExecutorImpl;
+import core.basesyntax.service.impl.ParserImpl;
+import core.basesyntax.service.impl.ReaderImpl;
+import core.basesyntax.service.impl.WriterImpl;
+import core.basesyntax.strategy.Operation;
+import core.basesyntax.strategy.impl.BalanceOperation;
+import core.basesyntax.strategy.impl.PurchaseOperation;
+import core.basesyntax.strategy.impl.ReturnOperation;
+import core.basesyntax.strategy.impl.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE = "src/main/resources/file.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/report.csv";
+
+    public static void main(String[] args) {
+        StorageDaoImpl storageDao = new StorageDaoImpl();
+        Map<FruitTransaction.Operation, Operation> operationsHandlers = new HashMap<>();
+        operationsHandlers.put(FruitTransaction.Operation.BALANCE,
+                new BalanceOperation(storageDao));
+        operationsHandlers.put(FruitTransaction.Operation.SUPPLY, new SupplyOperation(storageDao));
+        operationsHandlers.put(FruitTransaction.Operation.PURCHASE,
+                new PurchaseOperation(storageDao));
+        operationsHandlers.put(FruitTransaction.Operation.RETURN, new ReturnOperation(storageDao));
+        Reader reader = new ReaderImpl();
+        Parser parser = new ParserImpl();
+        OperationExecutor operationExecutor = new OperationExecutorImpl(operationsHandlers);
+        Writer writer = new WriterImpl();
+        List<String[]> lines = reader.read(INPUT_FILE);
+        List<FruitTransaction> transactions = parser.parse(lines);
+        operationExecutor.proceedAll(transactions);
+        writer.createReport(OUTPUT_FILE);
+    }
+}
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class (line 22) and the main method (line 26).